install.rb

```diff
@@ -86,7 +86,7 @@ def do_man(man, strip = 'man/')
 
       gzip = `which gzip`
       gzip.chomp!
-      `#{gzip} -f #{omf}`
+      `#{gzip} -fn #{omf}`
```

Comment:
I prefer the long options for cli commands. The are usually more meaningful and that makes it easier for other to review / read the code.

Reply:
Me too, but is it safe to assume the long option will exist on all operating systems (I think about AIX and other non-free UNIX)?

Reply:
I'll add-it as an extra commit, if it cause trouble it will be easy to roll back. :+1: